package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bcros-business-dashboard",
   "private": true,
   "type": "module",
-  "version": "1.1.19",
+  "version": "1.1.20",
```

Comment:
Updating the version number is helpful for tracking UI changes in the environments (Dev, Test, Prod), but if the only changes are to test files, it makes no difference.

However, you are changing actual UI code below, so leave this in this time.